test/functionalTest/cases/4666_kafka/kafka_notification_custom.test

```diff
@@ -0,0 +1,342 @@
+# Copyright 2025 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+#   General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+KAFKA custom notification tests
+
+--SHELL-INIT--
+dbInit CB
+
+# tracelevel 164 is the one for KAFKA notification logs
+kafkaCreateTopics "${KAFKA_BOOTSTRAP_A}" sub_E1 sub_E2
+brokerStart CB 164 IPV4
+accumulatorStart --pretty-print --bootstrapServers "${KAFKA_BOOTSTRAP_A}" --kafkaTopic "^sub_.*"
+
+--SHELL--
+
+#
+# Hint:  A Kafka cluster accessible at localhost:9092,localhost:9094 is required for this .test:
+#
+#
+# 01. Create KAFKA custom sub for entity E with topic sub_${id} payload A:{$A}
+# 02. Create entity E1 with A=1
+# 03. Create entity E2 with A=2
+# 04. Dump and reset accumulator, see: 1 KAFKA sub_E1 E A:1, 1 KAFKA sub_E2 A:2
+# 05. Update E1 with A=10
+# 06. Update E2 with A=20
+# 07. Dump and reset accumulator, see: 1 KAFKA sub_E1 E A:10, 1 KAFKA sub_E2 A:20
+# 08. GET subscriptions: 1 sub with timesSent 4 and lastSuccess
+#
+
+echo '01. Create KAFKA custom sub for entity E with topic sub_${id} payload A:${A}'
+echo "==========================================================================="
+payload='{
+  "description": "KAFKA sub",
+  "subject": {
+    "entities": [
+      {
+        "idPattern": ".*",
+        "type": "T"
+      }
+    ]
+  },
+  "notification": {
+    "kafkaCustom": {
+      "url": "kafka://localhost:9092,localhost:9094",
+      "topic": "sub_${id}",
+      "payload": "{ %22A%22: %22${A}%22 }"
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+echo
+echo
+
+
+accumulatorReset
```

Comment:
```suggestion
```

No need of accumulatorReset here, as the subscription has not been yet stimulated

Reply:
Fixed in c2c8b681a